_sass/testimonials.scss

```diff
@@ -13,3 +13,9 @@
     }
   }
 }
+
+.testimonial-footer {
+  color: $sr-blue;
+  font-family: "Open Sans Condensed", sans-serif;
+  text-transform: uppercase;
```

Comment:
This feels really easy to miss if we changed our heading styles. Maybe extract a heading style mixin and use that here?